packages/arb-token-bridge-ui/tests/e2e/specs/depositERC20.cy.ts

```diff
@@ -56,42 +49,42 @@ describe('Deposit ERC20 Token', () => {
         .should('have.text', 'ETH')
     })
 
-    it('should add ERC20 token correctly', () => {
-      // Click on the ETH dropdown (Select token button)
-      cy.findByRole('button', { name: 'Select Token' })
-        .should('be.visible')
-        .should('have.text', 'ETH')
-        .click({ scrollBehavior: false })
+    it('should bridge ERC-20 successfully', () => {
```

Comment:
Because of test isolation every `it` will have its own page load and wallet connection, which would make it very slow to do for every individual `it`.

That's why, where it makes sense, test cases are grouped together and `it` is replaced with `context`.

Eventually we can remove test isolation but it will require additional setup.